frontend/src/metabase-lib/lib/metadata/Database.ts

```diff
@@ -101,78 +107,38 @@ export default class Database extends Base {
   }
 
   // QUESTIONS
-  newQuestion() {
-    return this.question()
+  newQuestion(): Question {
+    return this.question({})
       .setDefaultQuery()
       .setDefaultDisplay();
   }
 
-  question(
-    query = {
-      "source-table": null,
```

Comment:
I'm using the types found in `metabase-types/types/Query` for now as I don't think we have anything newer or better for the data structures found in metabase-lib.

If you look at `StructuredQuery`, a `source-table` of `null` is not allowed but an undefined value is allowed. `DatasetQuery` expects a defined `query` property, so it seems like an empty object is the best resolution to this type error.